src/eddymotion/data/utils.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+import nibabel as nib
+import nitransforms as nt
+import numpy as np
+
+
+def apply_affines(nii, em_affines, output_filename=None):
+    """
+    Apply affines to supplied nii data
+
+    Parameters
+    ----------
+    nii : :obj:`Nifti1Image`
+        Nifti1Image data to be transformed
+    em_affines : :obj:`ndarray`
+        Nx4x4 array
+    output_filename : :obj:`str`, optional
+        String specifying filepath to which to save transformed Nifti1Image data
+
+    Returns
+    -------
+    nii_t_img : :obj:`Nifti1Image`
+        Transformed Nifti1Image data
+
+    """
+    transformed_nii = np.zeros(nii.shape)

```

Comment:
Let's not override the internal datatype of the original image.
```suggestion
    transformed_nii = np.zeros_like(np.asanyarray(nii.dataobj))

```